example/ios/Runner/AppDelegate.swift

```diff
@@ -10,62 +10,11 @@ import Branch
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?
```

Comment:
i got an error that told me to replace `UIApplicationLaunchOptionsKey` with `UIApplication.LaunchOptionsKey`, do you got it as well? i used the latest flutter version

Reply:
In Swift 4.2, `UIApplicationLaunchOptionsKey` has been renamed to `UIApplication.LaunchOptionsKey`.

Reply:
fix by https://github.com/blackmenthor/flutter-branch-io/pull/22/commits/35e096749cf87691009e2ce20235c5d9018a6174